etc/kayobe/kolla.yml

```diff
@@ -448,6 +456,12 @@ kolla_build_customizations_common:
     - /additions/*
   ironic_inspector_pip_packages_append:
     - /additions/*
+  neutron_server_packages_append:
+    - python3-libvirt
+    - python3-ethtool
+  nova_compute_packages_append:
```

Comment:
Annoyingly, this gets overridden by: [nova_compute_packages_append:](https://github.com/stackhpc/stackhpc-kayobe-config/pull/1102/files#diff-12d955d16a7bd8d6dc7bae8b56c2e485f4445434f0fb67546ea09c9e8b9b504dR481)

Due to https://github.com/stackhpc/stackhpc-kayobe-config/pull/1102/files#diff-12d955d16a7bd8d6dc7bae8b56c2e485f4445434f0fb67546ea09c9e8b9b504dR516

Reply:
The other thing we need is a release note